include/fi_enosys.h

```diff
@@ -150,7 +150,7 @@ int fi_no_atomic_compwritevalid(struct fid_ep *ep,
 /*
 static struct fi_ops_cm X = {
 	.size = sizeof(struct fi_ops_cm),
-	.getname = X,
+	.getname = fi_no_getname,
```

Comment:
I don't think the intent of what I had in mind came across clearly.  The intent is that these comments can be copied directly into a provider's source code.  The places where the functions are set to 'X' must be defined by the provider.  These functions are not optional.